doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst

```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
 The nRF7002 :term:`Expansion Board (EB)` (PCA63561), part of the `nRF70 Series Family <nRF70 Series product page_>`_, can be used to provide Wi-Fi® connectivity to compatible development or evaluation boards through the nRF7002 Wi-Fi 6 companion IC.
 For example, you can use it with the :ref:`Nordic Thingy:53 <ug_thingy53>`, an IoT prototyping platform from Nordic Semiconductor.
 
+You can also use the nRF7002 :term:`Expansion Board (EB)` (PCA63561) to provide Wi-Fi® connectivity to the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15`.
+
 .. figure:: images/nRF7002eb.png
    :alt: nRF7002 EB
 
    nRF7002 EB
 
+PCB edge connector and castellated edge holes
+*********************************************
+
 The nRF7002 EB features a :term:`Printed Circuit Board (PCB)` edge connector and castellated holes to provide Wi-Fi connectivity through the nRF7002 companion IC.
 
+The castellated holes on the side of the board allow the EB to be used as a breakout board that can be soldered to other PCB assemblies.
+This way, you can provide Wi-Fi capabilities to develop Wi-Fi applications with another System on Chip (SoC), MPU, or MCU host.
+For the pinout of the castellated holes, see `nRF7002 EB castellated edge holes`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
+
 In case of the Nordic Thingy:53, the PCB edge connector connects nRF7002 to nRF5340, which acts as a host.
 For the pinout of the PCB edge connector, see `nRF7002 EB PCB edge connector`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
 For information about how to connect to the Nordic Thingy:53, see `Using nRF7002 EB with the Nordic Thingy:53`_, also in the hardware guide.
 
-The castellated holes on the side of the board allow the EB to be used as a breakout board that can be soldered to other PCB assemblies.
-This way, you can provide Wi-Fi capabilities to develop Wi-Fi applications with another System on Chip (SoC), MPU, or MCU host.
-For the pinout of the castellated holes, see `nRF7002 EB castellated edge holes`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
```

Comment:
Please revert this change, so that the order mentioned in line 23 is kept in the order of paragraphs (edge connector first, castellated holes second).